hips/tiles/tile.py

```diff
@@ -0,0 +1,131 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
```

Comment:
Actually, this is required for the `HipsDescription` constructor, as it needs the properties file text.